querybook/webapp/redux/dataDocWebsocket/dataDocWebsocket.ts

```diff
@@ -58,6 +59,38 @@ export function openDataDoc(docId: number): ThunkResult<Promise<any>> {
                                 dataDoc,
                             },
                         });
+                    } else {
+                        // Even if it is sameOrigin, dervied fields also needs to be updated
```

Comment:
I dont really get why we need this part for same origin. Wouldn't the `!isSameOrigin` block be able to update the whole doc with derived fields?

Reply:
for sameOrigin, we only need to update meta_variables field. All other fields are already updated before the request locally is sent to server